suite-common/wallet-core/src/fiat-rates/fiatRatesThunks.ts

```diff
@@ -0,0 +1,379 @@
+import { differenceInMilliseconds, getUnixTime, subWeeks } from 'date-fns';
+
+import TrezorConnect, { AccountTransaction, BlockchainFiatRatesUpdate } from '@trezor/connect';
+import { createThunk } from '@suite-common/redux-utils';
+import { Network, networksCompatibility as NETWORKS } from '@suite-common/wallet-config';
+import { Account, CoinFiatRates, TickerId } from '@suite-common/wallet-types';
+import { FIAT } from '@suite-common/suite-config';
+import {
+    fetchCurrentFiatRates,
+    fetchCurrentTokenFiatRates,
+    fetchLastWeekRates,
+    getFiatRatesForTimestamps,
+} from '@suite-common/suite-services';
+import { getAccountTransactions, isTestnet } from '@suite-common/wallet-utils';
+import { getBlockbookSafeTime } from '@suite-common/suite-utils';
+
+import { actionPrefix } from '../accounts/constants';
+import { fiatRatesActions } from './fiatRatesActions';
+import { selectCoins } from './fiatRatesReducer';
+import { selectAccounts } from '../accounts/accountsReducer';
+import { INTERVAL, INTERVAL_LAST_WEEK, MAX_AGE, MAX_AGE_LAST_WEEK } from './constants';
+
+let staleRatesTimeout: ReturnType<typeof setInterval>;
+let lastWeekTimeout: ReturnType<typeof setInterval>;
+
+export const removeRatesForDisabledNetworks = createThunk(
```

Comment:
I would suggest adding `fiat` before `Rates` to this thunk name and also all other thunks, without it it's quite generic